rdf2vec/walkers/anonymous.py

```diff
@@ -4,10 +4,35 @@
 from hashlib import md5
 
 class AnonymousWalker(RandomWalker):
+    """Defines the anonymous walker of the walking strategy.
+
+    Attributes:
+        depth (int): The depth per entity.
+        walks_per_graph (float): The maximum number of walks per entity.
+
+    """
+
     def __init__(self, depth, walks_per_graph):
         super(AnonymousWalker, self).__init__(depth, walks_per_graph)
 
     def extract(self, graph, instances):
+        """Extracts a knowledge graph and transform it into a 2D vector, based
```

Comment:
"Extracts walks rooted at the provided instances which are then each transformed into a numerical representation".

In the "Returns" part I would then state: a 2D matrix (with number of rows = number of provided instances and number of columns = the embedding size)